exec/apk-gen.sh

```diff
@@ -43,6 +60,6 @@ then
 	if [ "$CIRCLE_BRANCH" == "$PUBLISH_BRANCH" ]; then
 		echo "Publishing app to Play Store"
 		gem install fastlane
-		fastlane supply --apk app-playStore-release.apk --track alpha --json_key ../exec/fastlane.json --package_name $PACKAGE_NAME
+		fastlane supply --apk susi-ai-master-app-playStore-release.apk --track alpha --json_key ../exec/fastlane.json --package_name $PACKAGE_NAME
```

Comment:
Let it be the old name

Reply:
@iamareebjamal I should just remove the "app" here or should I change the name in the above blocks too?

Reply:
Just here, no need to remove app